ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-release-3.0-tp1.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: maistra-builder
     namespace: ci
-    tag: upstream-1.22
```

Comment:
afaict we override this in every job to be `upstream-master`. we'd need to change that to actually be using this image. but: the 1.23 image is still not available, not sure why it hasn't been mirrored yet

Reply:
Fixing thanks